pkgs/applications/video/ani-cli/default.nix

```diff
@@ -22,16 +30,17 @@ stdenvNoCC.mkDerivation rec {
   };
 
   nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ gnugrep gnused curl fzf ffmpeg aria2 ]
```

Comment:
These are not build inputs.

Reply:
Is there a way to get the runtime dependencies when I create a shell for this package for example

```nix
{ pkgs ? import <nixpkgs> {} }:

pkgs.mkShell {
  inputsFrom = [ pkgs.ani-cli ];
}
```

that's the reason why I wanted to put them into the build inputs

Reply:
`mkShell` is not supposed to do that. The best you can do is to rename `buildInputs` to `runtimeDependencies`, then explicitly concat `pkgs.ani-cli.runtimeDependencies` with `[ pkgs.ani-cli ]` in your file.

Reply:
Ok thank you ^^